### DISCUSSION TITLE
Example strimzi cluster-operator config for json logs

### DISCUSSION THREAD

**Discussion by vutkin** [strimzi/strimzi-kafka-operator#6072] [Q&A]
Hi @scholzj , Is any example how could be configured strimzi cluster-operator to switch logs into json instead of console?

**Comment by scholzj** [comment]
There is no example. You can do it, but you basically have to configure the logging. You can check the system tests for it - maybe you can copy the stuff from there: https://github.com/strimzi/strimzi-kafka-operator/blob/main/systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java#L89
